### ISSUE TITLE
no function clause matching in Sentry.Event.culprit_from_stacktrace/1

### ISSUE THREAD

**Issue by atomkirk** [getsentry/sentry-elixir#313]
### Environment

* Elixir version (elixir -v): 1.7
* Erlang/OTP version (erl): 21.1
* Sentry version (mix deps): 7
* Operating system: Mac OS High Sierra

### Description
Got an error using `capture_exception`:

```
Elixir.FunctionClauseError: no function clause matching in Sentry.Event.culprit_from_stacktrace/1
  File "lib/sentry/event.ex", line 235, in Sentry.Event.culprit_from_stacktrace/1
  File "lib/sentry/event.ex", line 108, in Sentry.Event.create_event/1
  File "lib/sentry.ex", line 125, in Sentry.capture_exception/2
  File "lib/logging/exq_middleware.ex", line 73, in ZB.Exq.Middleware.Context.after_failed_work/1
    Sentry.capture_exception(
  File "lib/exq/middleware/pipeline.ex", line 79, in Exq.Middleware.Pipeline.chain/2
  File "lib/exq/worker/server.ex", line 119, in Exq.Worker.Server.handle_info/2
  File "gen_server.erl", line 637, in :gen_server.try_dispatch/4
  File "gen_server.erl", line 711, in :gen_server.handle_msg/6
  File "proc_lib.erl", line 249, in :proc_lib.init_p_do_apply/3
```

The arg was:
```
{:gen_server, :call, [#PID<0.3948.0>, {:checkout, #Reference<0.4264773998.1932787713.260773>, true, 10000}, 10000]}
```

so it looks like not all stacktraces are lists?

I guess it could be what exq is giving me in this callback:

```
  def after_failed_work(pipeline) do
    {reason, stacktrace} = pipeline.assigns.error

    Sentry.capture_exception(
      reason,
      stacktrace: stacktrace,
      extra: %{
        exq_job: pipeline.assigns.worker_module,
        job_serialized: pipeline.assigns.job_serialized
      }
    )

    pipeline
  end
```

**Comment by mitchellhenke** [comment]
Thanks for opening this!

A stacktrace is generally a list of 3 element tuples, as: `[{Module, Name, Arity}]`, so it's surprising to me that a timeout on a call would generate a `{reason, stacktrace}` where the stacktrace isn't a list.

I've confirmed this behavior (it seems to be related to the use of `spawn_link` instead of a GenServer), and will put together a fix.

**Comment by mitchellhenke** [comment]
This fix has been released in `7.0.3`
